vllm/attention/ops/nki_flash_attn.py

```diff
@@ -612,13 +827,19 @@ def flash_attn_varlen_nkifunc(
     n_kv_head=None,
     head_size=None,
     LARGE_TILE_SZ=2048,
-    return_debug_tensors=False,
     mixed_precision=True,
+    mask_reordered=True,
+    return_debug_tensors=False,
```

Comment:
to be more consistent with 

https://github.com/vllm-project/vllm/blob/4c822298981a8f7521492075ff72659985fc4c3f/vllm/v1/attention/backends/flash_attn.py#L197-L213 

remove both and set these internal argument inside the function call?

Reply:
Removed